ehsm_kms_service/function.js

```diff
@@ -34,6 +34,7 @@ const base64_encode = (str) => new Buffer.from(str).toString('base64')
 // base64 decode
 const base64_decode = (base64_str) =>
   new Buffer.from(base64_str, 'base64').toString()
+const is_base64 = (base64_str) => /^[a-zA-Z0-9\+\=\/]+$/gi.test(base64_str);

```

Comment:
length is 4X, (X > 0)  
= and == are at the ending

you could search from google.